src/sentry/issues/priority.py

```diff
@@ -82,6 +82,13 @@ def get_priority_for_escalating_group(group: Group) -> PriorityLevel:
     return PriorityLevel.HIGH
 
 
+def get_initial_priority(group: Group) -> PriorityLevel | None:
+    initial_priority = group.data.get("metadata", {}).get(
```

Comment:
May want to type this as `int | None` since python doesn't know what it is supposed to be. And below we probably want to return `PriorityLevel(initial_priority)`?